core/src/test/java/org/apache/calcite/test/FoodmartTest.java

```diff
@@ -38,6 +38,13 @@
 @Tag("slow")
 public class FoodmartTest {
 
+  private static final CalciteAssert.AssertThat ASSERT_FOODMART = CalciteAssert.that()
```

Comment:
I'm not sure what's the difference between `private static final` vs `@BeforeAll`, but I guess this is ok.

Reply:
It might be a bit safer to use `@BeforeAll` in combination with `@AfterAll`. I will change accordingly.